x-pack/platform/packages/shared/kbn-ai-assistant/src/hooks/use_knowledge_base.tsx

```diff
@@ -81,6 +94,32 @@ export function useKnowledgeBase(): UseKnowledgeBaseResult {
     [ml, service, notifications, statusRequest]
   );
 
+  const warmupModel = useCallback(
+    async (inferenceId: string) => {
+      setIsWarmingUpModel(true);
+      try {
+        await service.callApi('POST /internal/observability_ai_assistant/kb/warmup_model', {
+          params: {
+            body: {
+              inference_id: inferenceId,
+            },
+          },
+          signal: null,
+        });
```

Comment:
Let's align this with the `POST /internal/observability_ai_assistant/kb/setup` endpoint. I don't mind if  `inference_id` goes in the query or body params as long as it's the same.

Reply:
Another options it to add it to the path `POST /internal/observability_ai_assistant/kb/setup/{inference_id}`